pkg/apis/storage/validation/validation_test.go

```diff
@@ -91,6 +102,11 @@ func TestValidateStorageClass(t *testing.T) {
 			Provisioner: "kubernetes.io/foo",
 			Parameters:  longParameters,
 		},
+		"invalid reclaimpolicy": {
+			ObjectMeta:    metav1.ObjectMeta{Name: "foo"},
+			Provisioner:   "kubernetes.io/foo",
+			ReclaimPolicy: "Recycle",
```

Comment:
Recycle I am making invalid because it is deprecated, we shouldn't add any new functionality around it. Secondarily, NFS and hostPath are the only recyclable plugins and I can't think of any reason a provisioner would want to create Recyclable volumes (& introduce all the complications that come with it) when it can just Delete & Provision a new one.